postgres-external-storage/src/test/java/com/netflix/conductor/postgres/storage/PostgresPayloadStorageTest.java

```diff
@@ -47,6 +55,7 @@ public class PostgresPayloadStorageTest {
                     + " Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.";
     private final InputStream inputData;
     private final String key = "dummyKey.json";
+    private static AtomicInteger threadCounter = new AtomicInteger(0);
```

Comment:
Let's move the counter into each individual test so that the counter is not shared to prevent any side effects from 1 test to another